week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,7 +45,9 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+  
```

Comment:
You haven't written any code here. Did you commit the latest version of your code?

Reply:
Hi Sanyia,

Thank you for feedback its useful and I have prettier installed. I think the calculator tax returns correct.

I will do shakeball soon.

Rayhan